packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -157,6 +161,13 @@ class UnreferencedStateTracker {
     }
 }
 
+/**
+ * This sets the default container runtime expiry. We have container runtime expiry for GC purposes. Once the
```

Comment:
I am working on a README. I will create an issue and we can reference that here for now I think.

Reply:
You can add link to this issue here - https://github.com/microsoft/FluidFramework/issues/7478